src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh

```diff
@@ -8,8 +8,11 @@ mkdir /root/bin
 export CEPHADM_IMAGE='quay.ceph.io/ceph-ci/ceph:main'
 
 CEPHADM="/root/bin/cephadm"
+CEPHADM_SRC="/mnt/{{ ceph_dev_folder }}/src/cephadm/cephadm"
+
+# /mnt/{{ ceph_dev_folder }}/src/cephadm/build.sh $CEPHADM_SRC
+ln -s $CEPHADM_SRC $CEPHADM
 
-/mnt/{{ ceph_dev_folder }}/src/cephadm/build.sh $CEPHADM
 mkdir -p /etc/ceph
```

Comment:
i don't think changes in this file is necessary. we can probably keep it as it is and it should work